core/backend/package.json

```diff
@@ -130,7 +131,8 @@
     "js-base64": "^2.4.5",
     "multiparty": "^4.2.1",
     "semver": "^5.5.0",
-    "ws": "^7.2.0"
+    "ws": "^7.2.0",
+    "xmldom": "^0.5.0"
```

Comment:
Why is xmldom needed?

Reply:
`core/backend/src/IModelTransformer.ts` uses it for the default `onExportSchema` implementation which serializes the `ECSchemaMetaData.Schema` object. I couldn't find any other way to serialize that object. Or a way to export specific schemas from nativeDb. Is there any other example that shows how `onExportSchema` was intended to be overrided if the xml is needed to be read?

Reply:
Maybe this is a change request for the `ecschema-metadata` package then. Seems like there should be a `toString` or `writeToFile` method on the `Schema` class. Seems inefficient to create an XMLDom just to write out a file.